pyproject.toml

```diff
@@ -67,9 +66,9 @@ include = '''
     | /pinsys.py
     | /src/epyq/.*\.pyi?
     | /buildui.py
-    | /generate_versioning.py
 )$
 '''
+# include above used to have this: | /generate_versioning.py
```

Comment:
can delete

Reply:
Removed in 6d1cdf9cdfb1cd558a3c1d3896d3d7717e932c43